### DISCUSSION TITLE
pyproj error when importing scikit-mobility through Metaflow

### DISCUSSION THREAD

**Discussion by ekamperi** [pyproj4/pyproj#1107] [Q&A] [answered]
Hey everyone. Not sure if this is the right place to ask my question, so feel free to close it if I'm off. When I try to import scikit-mobility through a Metaflow-generated conda environment, I get the following error, *presumably* coming from pyproj:

![image](https://user-images.githubusercontent.com/9994264/178920444-276090a1-e22f-4ace-9f1e-ce306b7eb1ec.png)

Would you be so kind as to point me in directions I should look at to solve this? Thank you!

**Comment by ekamperi** [comment, marked as answer]
I tried exporting the full conda path up to "proj" under the PROJ_LIB env variable and it worked. Do you know why this is needed? If I open an interactive REPL and type import skmob, after having activated manually the conda environment, it works find even if PROJ_LIB is not set.

**Reply by snowman2** [reply]
When you activate your conda environment, it sets the `PROJ_LIB` environment variable.

**Reply by ekamperi** [reply]
Thank you for your prompt response! From that, I understand that it is Metaflow's fault that PROJ_LIB is not set.
